enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -140,6 +140,30 @@ class SubsidyAccessPolicy(TimeStampedModel):
     # ProxyAwareHistoricalRecords docstring for more info.
     history = ProxyAwareHistoricalRecords(inherit=True)
 
+    @property
+    def subsidy_active_datetime(self):
+        """
+        The datetime when this policy's associated subsidy is considered active.
+        If null, this subsidy is considered active.
+        """
+        return self.subsidy_client.retrieve_subsidy(subsidy_uuid=self.subsidy_uuid).get('active_datetime')
```

Comment:
nit: These subsidy getter lines are a bit long, maybe a helper method is in order?
```suggestion
        return self.get_subsidy_property('active_datetime')
```